examples/with_winit/src/lib.rs

```diff
@@ -168,7 +168,9 @@ struct VelloApp<'s> {
 
 impl<'s> ApplicationHandler<UserEvent> for VelloApp<'s> {
     #[cfg(target_arch = "wasm32")]
-    fn resumed(&mut self, _event_loop: &winit::event_loop::ActiveEventLoop) {}
+    fn resumed(&mut self, event_loop: &winit::event_loop::ActiveEventLoop) {
+        event_loop.set_control_flow(ControlFlow::Wait);
```

Comment:
The default is `Wait`, so you could also just not set it

Reply:
Oh neat! I'll change it once i'm available

Reply:
Thank u for the info! :D

Reply:
There, i think all of the suggestions are now applied :D